### ISSUE TITLE
Notconsole-tag evaluates as console

### ISSUE THREAD

**Issue by SvDvorak** [elastic/elasticsearch#23552]
While going through [getting started documentation](https://github.com/elastic/elasticsearch/blob/5.2/docs/reference/getting-started.asciidoc) I noticed a few issues relating to consoles with NOTCONSOLE-tags. It seems to evaluate as CONSOLE instead and where there is no PUT, POST, GET, etc in the console-text the copy link does nothing. [An example is the first curl call for getting elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/_installation.html#_installation).

Suggested solutions:

- [The first console in the following link](https://www.elastic.co/guide/en/elasticsearch/reference/current/_exploring_your_data.html#_loading_the_sample_dataset) uses NOTCONSOLE and expands into an incorrect curl-call. It probably instead should be edited to use PUT and GET with the CONSOLE-tag instead.
- The other usages of NOTCONSOLE are not needed it seems.

**Comment by dakrone** [comment]
@nik9000 I think this might be something you would be interested in.

**Comment by nik9000** [comment]
Thanks. I'll see about u breaking this.

**Comment by nik9000** [comment]
> Thanks. I'll see about u breaking this.

*un* breaking this....

**Comment by nik9000** [comment]
Fixed by https://github.com/elastic/docs/commit/a5331122575bf4e7543b11d41abdc32db4e38b79



> The other usages of NOTCONSOLE are not needed it seems.

They are, I think. This comes from the thing we are using to track our slow journey to 100% CONSOLE snippets. It's rules are:
0. If something is marked `NOTCONSOLE` it doesn't have to be converted no matter the rules below.
1. If the language is `sh` and the snippet contains `curl` then it must be converted.
2. If the language is `js` and the snippet isn't marked as `CONSOLE` then it must be converted.

The rest of the snippets in that doc are either `js` or have `curl` in them. At least that is the impression I got from scanning the file.
